### DISCUSSION TITLE
Safetensors deletion

### DISCUSSION THREAD

**Discussion by numix2** [lllyasviel/Fooocus#2785] [Q&A] [answered]
Is it fine to delete safetensors on it's own? I tried these so called "presets" and they downloaded the safetensors which are not small files. If I delete them will the program re-download it once I want to use it again? Or will it just break the whole thing?
I'm talking about the `\Fooocus_win64\Fooocus\models\checkpoints\` ones.

**Comment by DavidDragonsage** [comment, marked as answer]
Any base models or LoRAs that are installed by a preset will be reinstalled if you ever select that preset in the future.

Deletion of the base models in the checkpoints folder will not break Fooocus, they will just be reinstalled when necessary, i.e. if you select that preset again.
